server/s3_head_bucket_action.cc

```diff
@@ -61,15 +61,15 @@ void S3HeadBucketAction::send_response_to_s3_client() {
   if (reject_if_shutting_down() ||
       (is_error_state() && !get_s3_error_code().empty())) {
     S3Error error(get_s3_error_code(), request->get_request_id(),
-                  request->get_object_uri());
+                  request->get_bucket_name());
```

Comment:
why this is change? to match with AWS ?

Reply:
with object uri, we get "/" at the end of the resource or bucket name. This forward-slash is not there in the AWS response.

So I used get_bucket_name() instead of get_object_uri().